app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/destination/ChangeAddressSelectHousingTypeDestination.kt

```diff
@@ -70,43 +73,50 @@ private fun ChangeAddressSelectHousingTypeScreen(
       onDismiss = { onHousingTypeErrorDialogDismissed() },
     )
   }
+
+  uiState.errorMessage?.let {
+    ErrorDialog(
+      title = stringResource(hedvig.resources.R.string.general_error),
+      message = it,
+      onDismiss = { onErrorDialogDismissed() },
+    )
+  }
+
   HedvigScaffold(
     navigateUp = navigateUp,
   ) {
     Spacer(modifier = Modifier.height(48.dp))
     Text(
       text = stringResource(hedvig.resources.R.string.CHANGE_ADDRESS_SELECT_HOUSING_TYPE_TITLE),
-      style = MaterialTheme.typography.headlineSmall,
+      style = MaterialTheme.typography.headlineMedium,
       textAlign = TextAlign.Center,
       modifier = Modifier
         .fillMaxWidth()
         .padding(horizontal = 16.dp),
     )
-    Spacer(modifier = Modifier.height(64.dp))
+    Spacer(modifier = Modifier.weight(1f))
```

Comment:
Maybe also need some amount of space here for the "tiny screen" scenario?
```suggestion
    Spacer(modifier = Modifier.weight(1f))
    Spacer(modifier = Modifier.height(16.dp))
```